baseband/mark4/base.py

```diff
@@ -36,7 +36,7 @@ def read_frame(self, ntrack, decade=None, ref_time=None):
             approximate `ref_time`.
         ref_time : `~astropy.time.Time`, or None, optional
             Reference time within 4 years of the observation time.  Used only
-            if `decade` is ``None``.
+            if `decade` is `None`.
```

Comment:
maybe just `is not given`?

Reply:
I'm not certain about this.  If the user explicitly sets `decade=None` the same behaviour would result.